apps/coscrad-frontend/src/components/resources/factories/thumbnail-categorizable-detail-presenter-factory.tsx

```diff
@@ -14,7 +13,7 @@ import { VocabularyListDetailThumbnailPresenter } from '../vocabulary-lists/voca
 // TODO Define thumbnail specific presenters
 const lookupTable: { [K in CategorizableType]: FunctionalComponent } = {
     [CategorizableType.bibliographicReference]: BibliographicReferenceDetailThumbnailPresenter,
-    [CategorizableType.book]: BookDetailThumbnailPresenter,
+    [CategorizableType.book]: SongDetailThumbnailPresenter,
```

Comment:
Mistake?

Reply:
Ah! Yes I forgot to remove this. The build was breaking for Book in some strange way. There must be a circular dependency but I still need to track it down.